data/phones/phones/fre_phonemic.phones

```diff
@@ -1,42 +1,53 @@
-ʁ
-a
-i
-e
-s
-t
-l
-ɛ
-k
-ɔ
-m
-j
-n
+# CONSONANTS
 p
-d
-ɑ̃
 b
-ɔ̃
-y
-z
+m
 f
-ə
 v
-ɡ
+w
+t
+d
+n
+r  # Allophone of /ʁ/.
+s
+z
+l
+ʃ
 ʒ
+ɲ
+j
+ɥ
+k
+ɡ
+ŋ  # Only occurs in loanwords.
+ʁ
+# ORAL VOWELS
+i
+y
+e
+ø
+ɛ
+ɛː  # Allophone of /ɛ/ in European French.
+œ
+a
+ə
 u
+o
+ɔ
 ɑ  # Allophone of /a/ in European French.
+# NASALIZED VOWELS
 ɛ̃
-o
-ʃ
-w
-œ
-ø
-ɥ
-ɲ
-ŋ  # Only occurs in loanwords.
 œ̃  # Allophone of /ɛ̃/ in European French.
-ɛː  # Allophone of /ɛ/ in European French.
-r  # Allophone of /ʁ/.
-# The low tie bar is used in transcriptions to indicate that there is a morpheme boundary within a syllable as in "les uns les autres" /le.z‿œ̃ le.z‿otʁ/ 'each other (pl)' or "d'habitude" /d‿a.bi.tyd/ 'usually.' Strictly speaking, it should not be included in a phonemic transcription, since syllabification is not part of the underlying form. However, for many entries, the only transcription given includes tie bars, so the tie bar is necessary if we don't want to filter them out. Additionally, the tie bar is used when indicating the pronunciation of a normally silent consonant in liaison, as in "grand" /ɡʁɑ̃.t‿/.
+ɔ̃
+ɑ̃
+# The low tie bar is used in transcriptions to indicate that there is a 
+# morpheme boundary within a syllable as in 
+# *les uns les autres* /le.z‿œ̃ le.z‿otʁ/ 'each other (pl)' or 
+# *d'habitude* /d‿a.bi.tyd/ 'usually.' Strictly speaking, it should not be
+# included in a phonemic transcription, since syllabification is not part of
+# the underlying form. However, for many entries, the only transcription given
+# includes tie bars, so the tie bar is necessary if we don't want to filter 
+# them out. Additionally, the tie bar is used when indicating the 
+# pronunciation of a normally silent consonant in liaison, as in 
+# "grand" /ɡʁɑ̃.t‿/.
 ‿
-
```

Comment:
You deleted the newline here, is that intentional? Something your text editor is doing for you automatically?

Reply:
Huh, turns out it was my text editor doing that. Good to know.